javav2/example_code/lookoutvision/src/test/java/LookoutVisionTest.java

```diff
@@ -58,12 +97,334 @@ public void whenInitializingAWSRekognitionService_thenNotNull() {
+                projectName, modelVersion);
+
+        int index = tags.indexOf(tag);
+
+        assertEquals(-1, index);
+
+        System.out.println("Test 11 passed");
+
+    }
+
+    @Test
+    @Order(12)
+    public void describeModel_thenNotNull() throws IOException, LookoutVisionException {
+
+        // Describe a model.
+        ModelDescription description = Models.describeModel(lfvClient, projectName, modelVersion);
+
+        // Check that a description is returned.
+        assertNotNull(description);
+        System.out.println("Test 12 passed");
+
+    }
+
+    @Test
+    @Order(13)
+    public void listModels_thenNotNull() throws IOException, LookoutVisionException {
+
+        // Describe a model.
+        List<ModelMetadata> models = Models.listModels(lfvClient, projectName);
+
+        assertNotNull(models);
+
+        System.out.println("Test 13 passed");
+
+    }
+
+    @Test
+    @Order(14)
+    public void startModel_thenNotEquals() throws IOException, LookoutVisionException, InterruptedException {
+
+        // Describe a model.
+        ModelDescription model = Hosting.startModel(lfvClient, projectName, modelVersion, 1);
+
+        assertEquals(ModelStatus.HOSTED, model.status());
+
+        System.out.println("Test 14 passed");
+
+    }
+
+    @Test
+    @Order(15)
+    public void detectAnomaliesPanel_thenNotNull() throws IOException, LookoutVisionException {
+
+        DetectAnomalies panel = new DetectAnomalies(lfvClient, projectName, modelVersion, photo);
+        assertNotNull(panel);
+        System.out.println("Test 15 passed");
+
+    }
+
+    @Test
+    @Order(16)
+    public void stopModel_thenEquals() throws IOException, LookoutVisionException, InterruptedException {
+
+        // Describe a model.
+        ModelDescription model = Hosting.stopModel(lfvClient, projectName, modelVersion);
+
+        assertEquals(ModelStatus.TRAINED, model.status());
+
+        System.out.println("Test 16 passed");
+
+    }
+
+    @Test
+    @Order(17)
+    public void startModelPackagingJob_thenEqual() throws IOException, LookoutVisionException, InterruptedException {
+
+        // Create the model packaging job.
+        ModelPackagingDescription packageDescription = EdgePackages.startModelPackagingJob(lfvClient, projectName,
+                modelPackageJobJsonFile);
+
+        jobName = packageDescription.jobName();
+
+        assertEquals(ModelPackagingJobStatus.SUCCEEDED, packageDescription.status());
+
+        System.out.println("Test 17 passed");
+
+    }
+
+    @Test
+    @Order(18)
+    public void listModelPackagingJobs_thenEquals() throws IOException, LookoutVisionException, InterruptedException {
+
+        // List model packaging jobs.
+        List<ModelPackagingJobMetadata> packagingJobs = EdgePackages.listModelPackagingJobs(lfvClient, projectName);
+
+        // When run in sequence, jobName is picked up from
+        // startModelPackagingJob_thenEqual(). Otherwise specify a value for jobName.
```

Comment:
Add comma after Otherwise: 

Otherwise, specify a value for jobName.

Reply:
Fixed.